app/src/main/java/com/hsLink/hslink/data/dto/DummyDto.kt

```
@@ -0,0 +1,4 @@
+package com.hsLink.hslink.data.dto
+
+class DummyDto {
+}
```

detekt (1.23.8)

[warning] The class or object DummyDto is empty.

(detekt.empty-blocks.EmptyClassBlock)